neurovault/urls.py

```diff
@@ -409,6 +419,21 @@ def images(self, request, pk=None):
             page, context={'request': request}, many=True)
         return paginator.get_paginated_response(serializer.data)
 
+    def add_image(self, request, pk=None):
+        collection = get_collection(pk, request, mode='api')
+
+        if not owner_or_contrib(request, collection):
+            self.permission_denied(request)
+
+        image = StatisticMap(collection=collection)
+        serializer = EditableStatisticMapSerializer(data=request.data,
+                                                    instance=image)
```

Comment:
What is a proper way to differentiate between those other classes? Different endpoints is the simplest way to implement it. 
Another simple way is to require an additional argument `image_type` and to do something similar to this lines: https://github.com/NeuroVault/NeuroVault/blob/48743b44f04d37ab1e88ca8cc292ff19dc865be6/neurovault/urls.py#L134

What will be the most convenient way for a future user? How hard it'll be to guess the image type?